source/Conversation.cpp

```diff
@@ -114,6 +114,8 @@ void Conversation::Load(const DataNode &node, const string &missionName)
 			AddNode();
 			nodes.back().scene = SpriteSet::Get(child.Token(1));
 		}
+		else if(child.Token(0) == "invisible")
```

Comment:
I might sound a little bit close minded, but I would prefere having code changes separate from content/art

Reply:
They will be its just a dirty fix
When warp's npc actions will be implemented I wont need it anymore